frontend/packages/console-dynamic-plugin-sdk/package.json

```diff
@@ -15,9 +15,15 @@
     "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'",
     "publish": "yarn build && yarn publish dist --no-git-tag-version"
   },
+  "dependencies": {
+    "react-redux": "7.2.2",
```

Comment:
FYI, for now, we have lumped all dependencies (of any kind) into `devDependencies` but the upcoming update to plugin SDK package should address this.